app/build.gradle

```diff
@@ -1,174 +1,76 @@
-apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-allopen'
-apply plugin: 'kotlin-android-extensions'
-apply plugin: 'com.google.gms.google-services'
-apply plugin: 'com.google.firebase.crashlytics'
-apply plugin: 'com.jaredsburrows.license'
-apply plugin: 'kotlin-kapt'
-apply plugin: 'jacoco'
-
-String[] fileFilter = [
-        '**/*View*.*',
-        '**/*Activity.*'
-]
-enableJacoco(project, 'Debug', fileFilter)
-
-androidExtensions {
-    experimental = true
-}
-
-licenseReport {
-    generateHtmlReport = true
-    copyHtmlReportToAssets = true
-}
-
-allOpen {
-    annotation("com.karhoo.karhootraveller.util.annotation.OpenForTesting")
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
 }
 
 android {
-    testOptions {
-        unitTests.returnDefaultValues = true
-    }
-
-    configurations.all {
-        resolutionStrategy {
-            force 'com.google.code.findbugs:jsr305:1.3.9'
-        }
-    }
-
-    signingConfigs {
-        debug {
-            storeFile file("../tools/signing/debug.jks")
-            storePassword 'android'
-            keyAlias 'androiddebugkey'
-            keyPassword 'android'
-        }
-        staging {
-            storeFile file("../tools/signing/staging.jks")
-            storePassword 'karhooredlogo'
-            keyAlias "karhoo_qa"
-            keyPassword 'karhooredlogo'
-        }
-        prodQA {
-            storeFile file("../tools/signing/prodQA.jks")
-            storePassword 'karh00pink'
-            keyAlias "prodQA"
-            keyPassword 'karh00pink'
-        }
-    }
-
-    compileSdkVersion 29
-    buildToolsVersion '29.0.2'
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
 
     defaultConfig {
-        applicationId "com.karhoo.karhootraveller"
-        minSdkVersion 21
-        targetSdkVersion 29
-        versionCode 143
-        versionName "1.12.0"
-        testInstrumentationRunner "com.karhoo.karhootraveller.common.testrunner.TravellerTestRunner"
+        applicationId "com.karhoo.samples.uisdk.dropin"
+        minSdkVersion 22
```

Comment:
Has this been agreed? Assume it shouldn't be a problem but wasn't sure if this needs to be updated in any documentation?

Reply:
I guess this is for the sample app. we are not documenting this one. The one for the UI SDK perhaps need to be documented

Reply:
Do we want to keep the two in line wrt to versions supported?

Reply:
i left it as it was in the Dropin sample app, but i will lower it to 21